docs/content/version-reporting.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Privacy
+kind: operations
+weight: 50
+---
+
+This document provides details about OPA's anonymous version reporting feature.
+
+## Overview
+
+OPA periodically reports its version to a publicly hosted, external service. The version reports only contain the OPA
+version number (e.g., v0.12.3) and a randomly generated UUID. This feature is only applicable to the `opa run` and
+`opa version` commands.
+
+In case of the `opa run` command, this feature is **ON by-default** and can be easily disabled by specifying
+the `--incognito` flag. When OPA is started in either the server or repl mode, OPA calls the external service
+on a best-effort basis and shares the version it's running. The time taken to execute the remote call and process the
+subsequent response from the external service does not delay OPA's start-up.
```

Comment:
In addition to not delaying, it might be a good idea not to log any communication errors (timeouts in particular) to the telemetry service. That is, we don't want global OPA fleet to log errors due to some transient issues with the telemetry service.

Reply:
In `repl` mode: A message will be shown only when there's an update. 
In `server` mode: When a new version is available it will logged at INFO. Other messages are logged at DEBUG.